typescript/test/testdata/echo_client_baseline.ts.txt

```diff
@@ -227,4 +229,33 @@ export class EchoClient {
     options = options || {};
     return this._innerApiCalls.echo(request, options, callback);
   }
+  expand(
+      request?: protos.google.showcase.v1beta1.IExpandRequest,
+      options?: gax.CallOptions):
+    gax.ServerStreamingCall{
+    request = request || {}; 
+    options = options || {};
+    return this._innerApiCalls.expand(request, options);
+  }
+  collect(
+      options: gax.CallOptions|Callback<
```

Comment:
It seems there should be two overloads here:
```
collect(options: gax.CallOptions, callback: Callback<......>);
collect(callback: Callback<.......>);
```
and the function definition would then be
```
collect(optionsOrCallback: gax.CallOptions|Callback<......>, callback?: Callback<......>) {
   ...
}
```

Reply:
The difference is that the current overloads allow calling `collect(callback, options)` (reversed order) and it should be forbidden.